edk2toollib/windows/policy/firmware_policy.py

```diff
@@ -0,0 +1,561 @@
+# @file
+#
+# Library to parse/manage/build unsigned firmware policy blobs
+#
+# Copyright (c), Microsoft Corporation
```

Comment:
Inconsistent copyright formatting.

Reply:
Fix pending for next commit.
**NOTE:** I copied/pasted from MU_BASECORE\BaseTools\Edk2ToolsBuild.py ... [line 86](https://github.com/microsoft/mu_basecore/blob/d7307756c69c6da1a7aaa554441bf431d2694501/BaseTools/Edk2ToolsBuild.py#L86).  Shall I fix that too?

Reply:
Fixed in latest commit